### DISCUSSION TITLE
Bromite repo latest updates showing 3 years old in F-droid

### DISCUSSION THREAD

**Discussion by ghost** [bromite/bromite#1772] [Q&A]
Device OS and version: MIUI GLOBAL 12.5.1 **STABLE.**

Device model/manufacturer: **REDMI NOTE 7 S (LAVENDER).**

F-Droid version (in the About screen): **1.14**

F-Droid privileged extension version (if installed): **NO**

What did you expect to see?

_EXPECTED TO SEE THE CORRECT DATE WHEN THE LATEST UPDATES OF THE APPS WERE PUBLISHED_

What did you see?

_THE DATE ON WHICH THE LATEST UPDATES OF BROMITE REPO WERE PUBLISHED WAS SHOWING 20/01/1970 AND AT THE TOP IT SHOWED "3 YEARS AGO"._
![pic1](https://user-images.githubusercontent.com/88554859/152725595-e5404d1a-d205-4aae-9341-09ab7cf7e9df.jpg)
![pic2](https://user-images.githubusercontent.com/88554859/152725598-d9b7b5bf-5de5-40b3-8a29-4af608d2dce2.jpg)
![pic3](https://user-images.githubusercontent.com/88554859/152725600-86d6e087-6471-4ca9-b088-7a680f5b52c4.jpg)

**Comment by ghost** [comment]
I have created a new issue but it was closed as I did not maintain the proper template ... Guys would you please check it out what happened. Is it a problem of "Bromite repositor"y or "F-Droid" itself?

**Reply by csagan5** [reply]
It's the Bromite repository.
